notebook-lm-clone/src/podcast/text_to_speech.py

```
@@ -0,0 +1,203 @@
+        
+        logger.info(f"Podcast generation complete! Generated {len(output_files)} files")
+        return output_files
+    
+    def _generate_single_segment(self, speaker: str, text: str) -> Any:
+        voice = self.speaker_voices.get(speaker, "af_heart")
+        clean_text = self._clean_text_for_tts(text)
+
+        generator = self.pipeline(clean_text, voice=voice)
+        
+        combined_audio = []
+        for i, (gs, ps, audio) in enumerate(generator):
```

Ruff (0.13.1)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)